client/src/GlobalStyles.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Global, css } from '@emotion/core';
+
+function GlobalStyles() {
+  return (
+    <Global
+      styles={(theme) => css`
+        @import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&display=swap');
```

Comment:
Good solution to move the font import to `GlobalStyles` as now all global styles can be found in one file. 👍